application/auth/views.py

```diff
@@ -1,19 +1,34 @@
-from flask import Blueprint, render_template, jsonify
-from application.auth.helpers import (token_required, update_user)
+from flask import Blueprint, render_template, jsonify, session, request
+from application.auth.helpers import (update_user)
 
 auth = Blueprint('auth', __name__, template_folder='templates')
 
 
 @auth.route('/')
 def login():
     context = dict()
-    context["title"] = "Dashboard"
+    context["title"] = "API-Monitor | Login"
+    session.clear()
     return render_template("auth/login.html", context=context)
 
 
 @auth.route('/auth', methods=['POST'])
-@token_required
-@update_user
 def authenticate():
-    return jsonify(dict(status="success", message="login successful"))
+    try:
+        payload = request.get_json()
+        update_user(payload)
```

Comment:
Doesn't this payload need to be checked before being used?

Reply:
Not really, by the time you access the payload, it already has user data